packages/babylon/src/plugins/flow.js

```diff
@@ -2267,4 +2268,18 @@ export default (superClass: Class<Parser>): Class<Parser> =>
         /* isAsync */ true,
       );
     }
+
+    skipFlowComment(): ?number {
+      const includeComment =
+        this.input.charCodeAt(this.state.pos + 2) === charCodes.colon &&
+        this.input.charCodeAt(this.state.pos + 3) === charCodes.colon &&
+        4; // check for /*::
```

Comment:
What is this `&& 4` for?

Reply:
for ``/:**``, we have to advance four steps